src/features/register/components/CheckoutButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Button, ButtonProps, Spinner } from "reactstrap";
+import { Shirt } from '../models';
+
+const stripePromise = loadStripe(import.meta.env.VITE_STRIPE_PUBLIC);
+
+interface CheckoutButtonProps extends ButtonProps {
+  registration: string,
```

Comment:
```suggestion
  registration: "registration-fee-solo" | "registration-fee-team" | "funktion" | "bomull" | "keps"
```

Eller gör en typ av det som vi kan återanvända

Reply:
Jag veeegrar
just nu